Coordinates coordinates
CovarianceMatrix covariance_matrix
ConeColor[] cone_colors
float32 cone_probability
